inc/switchtec/log.h

```diff
@@ -46,7 +46,8 @@ struct log_a_retr_result {
 	struct log_a_retr_hdr {
 		uint8_t sub_cmd_id;
 		uint8_t from_end;
-		uint8_t reserved1[2];
+		uint8_t overflow;
+		uint8_t reserved1;
```

Comment:
Push this into the previous commit and probably add a comment to say overflow only applies with the _FLAGS subcmd....

Though I question why a new subcmd ID was even necessary in the firmware? Can't it just start setting the reserved field or not? Presumably firmware that doesn't support will just leave the reserved field zeroed.

Reply:
Moved to previous commit along with comments.

Yes it seems overkill to add new subcommands for this purpose, but it's the hardware convention so we will follow the implementation.